src/items.cpp

```diff
@@ -1606,13 +1591,12 @@ void consumeItem(Item*& item, int player)
 	{
 		if ( item->node != nullptr )
 		{
-			int i;
-			for ( i = 0; i < MAXPLAYERS; i++ )
+			for ( auto* const stat : stats )
```

Comment:
again, should be left as standard for loop instead of auto

Reply:
Fixed in f76c857